index.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="index.css">
+  <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.5.0/css/all.css"
+      integrity="sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU"
+      crossorigin="anonymous"
+    />
+  <title>NeoDev Store</title>
+</head>
+<body>
+  <div class="homepage_container">
+  <div class="navigation_main_container">
+    <div class="nav_section">
+      <div class="burger">
+        <i class="fas fa-bars"></i>
+      </div>
+      <div class="brand_logo">
+        <a href="/">
+          <img src='./images/logo.png' alt="" />
```

Comment:
You can add a GitHub raw URL for the image so it won't have to look into the folder to get the image it will just get it from the url

Reply:
okay.